examples/plot_2_multidim_sigs.py

```diff
@@ -0,0 +1,249 @@
+  recording.
+- A 1D list of feature dataframes is returned.
+
+For 3d arrays:
+
+- Use ``axis = 0`` (default) when computing features independently across the first axis.
+  This is likely the case for a (n_channels, n_epochs, n_timepoints) organization.
+- Use ``axis = 1`` when computing features independently across the second axis. A possible
+  organization is (n_epochs, n_channels, n_timepoints).
+- Use ``axis = (0, 1)`` when computing features independently, for each signal, across both axes.
+  This may be the case for an (n_subjects, n_channels, n_timpoints) oraganization
+- Use ``axis = None`` when each signal is dependent on others. This may be the case for an
+  (n_runs, n_epochs, n_timpoints) oraganization, assuming a continuous recording.
+- A 2D list of feature dataframes is returned.
+
+"""
+
+####################################################################################################
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from neurodsp.sim import sim_combined
+from bycycle.group import compute_features_2d, compute_features_3d
+from bycycle.plts import plot_feature_categorical, plot_feature_hist
+
+####################################################################################################
+
+# Simulation settings
+n_seconds = 10
+fs = 500
+
+freq = 10
+f_range = (5, 15)
+
+epochs = 10
+channels = 5
+
+####################################################################################################
+
+####################################################################################################
+#
+# Example 1. 2D Array (n_epochs, n_timepoints)
+# --------------------------------------------
+# The features for a 2d array with a (n_epochs, n_timepoints) organiation will be computed here.
+# This is an example of when using ``axis = None`` is appropriate, assuming each epoch was recorded
+# continuously. Data will be simulated to produce to alternating epoch types (fixation and task)
+# with varying rise-decay symmetries.
+
+# Simulate a timeseries
+sigs = np.zeros((epochs, n_seconds*fs))
+
+for idx in range(epochs):
+
+    if idx % 2 == 0:
+        # Fixation epoch
+        sim_components = {'sim_powerlaw': dict(exponent=-2),
+                          'sim_bursty_oscillation': dict(freq=10, cycle='asine',
+                                                         enter_burst=0.75, rdsym=0.5)}
+    else:
+        # Task epoch
+        sim_components = {'sim_powerlaw': dict(exponent=-2),
+                          'sim_bursty_oscillation': dict(freq=10, cycle='asine',
+                                                         enter_burst=0.75, rdsym=0.75)}
+
+    sigs[idx] = sim_combined(n_seconds, fs, components=sim_components)
+
+####################################################################################################
+
+# Compute features
+thresholds = dict(amp_fraction_threshold=0., amp_consistency_threshold=.5,
```

Comment:
In a tutorial, we might want to add a note about where these setting values come from. Are they chosen as defaults?